fractal_tasks_core/lib_ngff.py

```diff
@@ -388,7 +399,15 @@ def load_NgffImageMeta(zarr_path: str) -> NgffImageMeta:
     Returns:
         A new `NgffImageMeta` object.
     """
-    zarr_group = zarr.open_group(zarr_path, mode="r")
+    try:
+        zarr_group = zarr.open_group(zarr_path, mode="r")
+    except GroupNotFoundError as e:
+        error_msg = (
+            f"\nOriginal error: {str(e)}.\n"
+            f"Zarr group not found at {zarr_path}."
```

Comment:
Suggestion: f"Could not load the requested image, because no Zarr image was found at {zarr_path}"

Reply:
Updated to
```python
f"Could not load attributes for the requested image, because no Zarr image was found at {zarr_path}"
```